charts/passbolt/templates/ingress.yaml

```diff
@@ -26,16 +26,20 @@ spec:
       secretName: {{ .secretName }}
     {{- end }}
   {{- end }}
+  ingressClassName: {{ .Values.ingress.className }}
```

Comment:
`ingressClassName` is not required if it's defined on the cluster level.

You should surround this line with a condition like
```
{{- if .Values.ingress.ingressClassName }}
```
in order to use the ingress class by default